wowup-electron/src/app/pages/get-addons/get-addons.component.ts

```diff
@@ -156,6 +156,11 @@ export class GetAddonsComponent implements OnInit, OnDestroy {
       return;
     }
 
+    if (this._addonService.getEnabledAddonProviders().length === 0) {
+      this._displayAddonsSrc.next([]);
+      return;
+    }
```

Comment:
Not sure if this is the best way to ensure no addons are shown in the get-addons page.

The problem is that when there are no sources, the `next` on the Observer is never called and it stays on "loading" forever due to the busy flag. If I add a `success` to remove the busy flag, it will keep the old contents because the `_displayAddonsSrc` is never updated.